### PR TITLE
ZEPPELIN-964 ] Jobmanagement - (2) basic front-end

### PR DESCRIPTION
**Pull request by cloverhearts**
### What is this PR for?
Job manger basic front end.
You can check to paragraph and notebook information.
It is created with the following additional functions PR.
(E. G., Filter and sort)

this PR is divided from the https://github.com/apache/zeppelin/pull/921

### What type of PR is it?
Featrue

### Todos
- [x] - notebook information
- [x] - viewing status for paragraph
- [x] - feat. running progress bar.
- [x] - added job menu in navbar. 
(It can be modified by the following (https://github.com/apache/zeppelin/pull/1013)

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-964

### How should this be tested?
1. create and running, modified notebook and paragraph.
2. check to cron or normal notebook status in job manager.

### Screenshots 

job manger-basic

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes

### CODE REVIEW

**Comment by astroshim** [comment]
The job page looks greate and working good.
I think it seems to be better if the Job description is changed. :)
Then how can i control the the notebook action in the Job page?

**Comment by cloverhearts** [comment]
@astroshim 
This PR is unable to control the Notebook.
Notebook control is ready for the next PR.

**Comment by cloverhearts** [comment]
@astroshim 
check to sub-task section.
https://issues.apache.org/jira/browse/ZEPPELIN-531
Thank you.

**Comment by astroshim** [comment]
@cloverhearts Thanks for the quick response. I understand. :+1:

**Comment by AhyoungRyu** [comment]
@cloverhearts Great feature indeed! 👍
It's minor thing but, it seems the first tooltip is not shown properly.

screen shot 2016-06-20 at 12 37 42 pm

**Comment by cloverhearts** [comment]
@AhyoungRyu 
Thank you for you feedback.
I'll fix that

**Comment by cloverhearts** [comment]
merge from master branch and replace job menu location.

**Comment by Leemoonsoo** [comment]
IMO, 'Job' menu item can stay next to 'Notebook', instead of going into dropdown menu.
Because, "Interpreter', "Credential", "Configuration" menus are related configuring Zeppelin, and "Notebook", "Job" are related to using Zeppelin.

**Comment by cloverhearts** [comment]
@Leemoonsoo 
Thank you for your feed back.
You are right.
I have completed modifications.


and
@AhyoungRyu  fixed position for tooltip (top -> top-left)

**Comment by jongyoul** [comment]
ping

**Comment by jongyoul** [comment]
Is this PR completed?

**Comment by cloverhearts** [comment]
@jongyoul Yes, Completed.

**Comment by jongyoul** [comment]
It remains for the review, doesn't it?

**Comment by cloverhearts** [comment]
This PR is now completed.
In my opinion, it seems there is no problem.
Please merge after review.

File: zeppelin-web/src/app/jobmanager/jobs/job.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: gray" class="fa fa-circle"
+           tooltip-placement="top-left"
+           tooltip="{{paragraphJob.name}} is PENDING">
+        </i>
+      </a>
+      <a ng-switch-when="RUNNING"
+         style="text-decoration: none !important;"
+         ng-href="#/notebook/{{notebookJob.notebookId}}/paragraph/{{paragraphJob.id}}">
+        <i style="color: blue" class="fa fa-spinner spinAnimation"
```
Comment (Leemoonsoo) [inline zeppelin-web/src/app/jobmanager/jobs/job.html]: In Chrome, it does not animate 'fa-spinner'. How about `fa fa-spinner fa-spin` to animate in chrome?

**Comment by Leemoonsoo** [comment]
Notebook list order in Job menu is different from notebook list in homepage and menu bar.

Notebook list in home page
image

Notebook list in job menu
image

If it list notebook in same order, it'll be much easier to find notebook in job menu.

File: zeppelin-web/src/app/jobmanager/jobs/job.html (hunk 1 above)
Comment (cloverhearts) [inline zeppelin-web/src/app/jobmanager/jobs/job.html]: @Leemoonsoo 
Add css code for the chrome browser
Thank you for your feedback.

spin
chrome browser.

**Comment by cloverhearts** [comment]
@Leemoonsoo 
It seems you want to talk about sort Notebook name.
I dropped implemented it already, but it will be merged with the next step of the PR.
Attach the related links.

- Jira issue (filter and name sort)
https://issues.apache.org/jira/browse/ZEPPELIN-967

- Code (old PR)
https://github.com/apache/zeppelin/pull/921/files#diff-10471e430df1559157f6044bac4ca93dR44

**Comment by Leemoonsoo** [comment]
Thanks @cloverhearts for the new feature.
CI failure unrelated. 
LGTM and merge if there're no more discussion.

**Pull request closed without merging**